movies/moviesTests/Core/Controller/TabBarViewControllerTestCase.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  TabBarViewControllerTestCase.swift
+//  moviesTests
+//
+//  Created by gustavo.cosenza on 07/07/22.
+//
+
+import XCTest
+@testable import movies
+
+class TabBarViewControllerTestCase: XCTestCase {
+
+    var sut: TabBarViewController!
+    
+    override func setUp() {
+        super.setUp()
+        sut = TabBarViewController()
+    }
+
+    override func tearDown() {
+        sut = nil
+        super.tearDown()
+    }
+
+    func testSetup() {
+        XCTAssertEqual(sut.tabBar.isTranslucent, false)
```

Comment:
Esse teste não gera muito valor. UI é muito volátil para ser validada como teste unitário.

Reply:
na verdade eu tive que trocar a minha máquina e acabei subindo o que ja tinha começado hauhuah
entendo ele ser volátil, mas pra esse caso ele não poderia mudar, então eu deixei assim pra forçar ele não ser alterado... mas também não sei uma forma melhor auuauahhau